example/android/app/src/androidTest/java/com/example/InstabugSample/util/InstabugAssertions.java

```diff
@@ -0,0 +1,19 @@
+package com.example.InstabugSample.util;
+
+import static androidx.test.platform.app.InstrumentationRegistry.getInstrumentation;
+import static org.junit.Assert.assertTrue;
+
+import androidx.test.uiautomator.UiDevice;
+import androidx.test.uiautomator.UiObject;
+import androidx.test.uiautomator.UiSelector;
+
+public class InstabugAssertions {
+    private static final UiDevice device = UiDevice.getInstance(getInstrumentation());
+
+    public static UiObject assertViewWillBeVisible(String resourceId, long timeout) {
```

Comment:
**suggestion:** `assertViewIsVisible`/`assertIsVisible` seems to be a better name, keeping the fact that the view might be visible already by the time this is called. What do you think?

Reply:
I added `well` in there just to make it obvious that its use is for views that don't just appear immediately.